libs/commons-database/src/main/kotlin/gov/cdc/ocio/database/couchbase/CouchbaseCollection.kt

```diff
@@ -0,0 +1,90 @@
+package gov.cdc.ocio.database.couchbase
+
+import gov.cdc.ocio.database.persistence.Collection
+import com.couchbase.client.java.Scope
+import com.couchbase.client.java.json.JsonObject
+import com.google.gson.*
+import gov.cdc.ocio.database.utils.DateLongFormatTypeAdapter
+import gov.cdc.ocio.database.utils.InstantTypeAdapter
+import java.time.Instant
+import java.util.*
+
+
+/**
+ * Couchbase Collection implementation
```

Comment:
`@param` statement for collectionName, `@property` statement for couchBaseScope and `@return` statement are missing from `Kdoc`.

Reply:
Fixed